api/src/middleware/team.js

```diff
@@ -23,6 +24,45 @@ async function validateTeamId(req, res, next) {
   next();
 }
 
+async function validateTeamRepo(req, res, next) {
+  const { ghToken } = req.body;
+  const { data } = await axios.get('https://api.github.com/user',
+    {
+      headers: { Authorization: `token ${ghToken}` },
+    });
```

Comment:
What happens if this errors out (eg invalid token)? Do you need a try/catch block here to handle it?